pkg/controller/pod.go

```diff
@@ -131,6 +139,25 @@ func (pnc *PodNetworksController) worker() {
 	}
 }
 
+func (pnc *PodNetworksController) reconcileOnStartup() error {
+	pods, err := pnc.podsLister.List(labels.Everything())
```

Comment:
shouldn't we limit this to pod's running on the node where this particular instance is running, like done in https://github.com/k8snetworkplumbingwg/multus-dynamic-networks-controller/pull/90/files#diff-723add03c98864a94589c5e1e0dc8c5a47960f3d50173ed24642ba2c12ccff10R134 ?

Reply:
Ah of course. Thank you for pointing out that fact. Makes sense.